src/generated/resources/data/occultism/recipe/ritual/possess_random_animal_special.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "occultism:ritual",
+  "activation_item": {
+    "item": "occultism:book_of_binding_bound_afrit"
+  },
+  "duration": 60,
+  "entity_tag_to_summon": "occultism:random_animals_special",
+  "ingredients": [
+    {
+      "item": "minecraft:white_wool"
+    },
+    {
+      "item": "minecraft:light_gray_wool"
+    },
+    {
+      "item": "minecraft:gray_wool"
+    },
+    {
+      "item": "minecraft:black_wool"
+    },
+    {
+      "item": "minecraft:brown_mushroom_block"
```

Comment:
Similarly, what does forcing the block here add over simply having them supply a mushroom?